workers/request-worker.js

```diff
@@ -111,7 +111,10 @@ function makeRequest (job, callback) {
         koop.Cache.insertPartial('agol', job.data.itemId, geojson, job.data.layerId, function (err) {
           if (err) {
             featureService.pageQueue.kill()
-            throw err
+            koop.log.error('err')
+            var error = new Error('Error inserting rows into the db')
+            error.type = 'db'
+            throw error
```

Comment:
Noticing we're just straight up throwing errors in three places here. Is that not an issue?